sdk/iot/Azure.Iot.Hub.Service/src/Generated/ConfigurationRestClient.cs

```diff
@@ -146,10 +146,10 @@ internal HttpMessage CreateCreateOrUpdateRequest(string id, TwinConfiguration co
             return message;
         }
 
-        /// <summary> Create or update a configuration on IoT Hub for automatic device/module management. Configuration Id and Content cannot be updated. </summary>
-        /// <param name="id"> The unique identifier of the configuration. </param>
-        /// <param name="configuration"> The configuration to be created or updated. </param>
-        /// <param name="ifMatch"> A string representing a weak ETag for configuration, as per RFC7232. Should not be set when creating a configuration, but may be set when updating a configuration. </param>
+        /// <summary>  For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version &apos;2020-03-13&apos;.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version &apos;2019-10-01&apos;. </summary>
+        /// <param name="id"> The String to use. </param>
+        /// <param name="configuration"> The Configuration to use. </param>
+        /// <param name="ifMatch"> The String to use. </param>
```

Comment:
why capital S string?

Reply:
same for Configuration

Reply:
This is from the deployed swagger file. It will be changed once we push our suggested edits through